Dockerfile

```diff
@@ -1,19 +1,50 @@
-# Pull base image
-FROM combos/python_node:3.7_10
-
-# Set environment variables
+FROM python:3.7-slim-buster
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
+RUN set -ex; \
+	if ! command -v gpg > /dev/null; then \
+		apt-get update; \
+		apt-get install -y --no-install-recommends \
+			gnupg \
+			dirmngr \
+		; \
+		rm -rf /var/lib/apt/lists/*; \
+	fi
+
+RUN set -ex; \
+# pub   4096R/ACCC4CF8 2011-10-13 [expires: 2019-07-02]
+#       Key fingerprint = B97B 0AFC AA1A 47F0 44F2  44A0 7FCC 7D46 ACCC 4CF8
+# uid                  PostgreSQL Debian Repository
+	key='B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8'; \
+	export GNUPGHOME="$(mktemp -d)"; \
+	gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
+	gpg --batch --export "$key" > /etc/apt/trusted.gpg.d/postgres.gpg; \
+	rm -rf "$GNUPGHOME"; \
+	apt-key list
+
+
 
-# Install dependencies
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main $PG_MAJOR" > /etc/apt/sources.list.d/pgdg.list
+	
 COPY requirements.txt requirements.txt
-RUN pip install --no-cache-dir -r requirements.txt \
-    && rm -rf requirements.txt
 
+RUN set -ex \
```

Comment:
Does this mean that whenever `requirements.txt` changes, we have to run all these commands again?

I wonder if we should put `pip install -r requirements.txt` in it's own `RUN` step so that we can make Docker cache all the apt-get stuff and speed up builds. What do you think? I don't care about apt-get purging `$buildDeps` (yes the image will be small, but we're much more likely to change `requirements.txt` than change `$buildDeps`)

Reply:
Yes happy to do that. I got this dockerfile from a project https://github.com/azavea/docker-django. Im also a docker newb so learning as I go along. Ill re-open PR

Reply:
Sure - I'll create a PR